src/logic/function/basic.lean

```diff
@@ -17,6 +17,12 @@ namespace function
 section
 variables {α : Sort u} {β : Sort v} {f : α → β}
 
+/-- Evaluation a function to an argument. Useful if you want to talk about the partially applied
+  `function.eval x : (Π x, β x) → β x`. -/
```

Comment:
```suggestion
/-- Evaluate a function at an argument. Useful if you want to talk about the partially applied
  `function.eval x : (Π x, β x) → β x`. -/
```

Reply:
I'm mildly in favour of making `eval` in all the various namespaces a `protected def`. What do you think?